babel.config.js

```diff
@@ -9,4 +9,13 @@ module.exports = {
       plugins: ['@babel/plugin-transform-modules-commonjs'],
     },
   },
+  plugins: [
+    [
+      'babel-plugin-styled-components',
+      {
+        ssr: true,
+        fileName: false,
+      },
+    ],
+  ],
```

Comment:
I've tried removing this locally and it doesn't seem to break anything - why was it added?

Reply:
@ChrisBAshton 
https://www.styled-components.com/docs/tooling#babel-plugin
https://github.com/BBC-News/simorgh/blob/latest/.babelrc#L28

Reply:
I also don't need it

Reply:
Thanks @radiocontrolled. Let's leave it out as you've done for now, we can discuss when we refine https://github.com/BBC-News/psammead/issues/159